internal/pkg/server/server.go

```diff
@@ -40,6 +48,7 @@ func NewServer(opt ServerOpts) *Server {
 	})
 	g.POST("/tar", s.postTar)
 	g.GET("/tar", s.getTar)
+	g.POST("/helm", s.postHelm)
```

Comment:
I would prefer `/PLUGIN/tar` -> `/helm/tar` this would allow to specify the format separately if the need arises.